http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -416,4 +422,26 @@ private static boolean hasScope(String scopeParam, String targetScope) {
         }
         return false;
     }
+
+    private String combineAndReorderScopes(Set<String> allScopes, String paramValue) {
+        StringBuilder combinedScopes = new StringBuilder();
+        addScope(paramValue, allScopes);
+
+        if (allScopes.contains(OIDC_SCOPE)) { //some OpenID providers require openid scope to be added in the beginning
+            allScopes.remove(OIDC_SCOPE);
+        }
+        combinedScopes.append(OIDC_SCOPE);
+        for (String scope : allScopes) {
+            if (scope != null && !scope.isEmpty()) {
+                combinedScopes.append(" " + scope);
```

Comment:
@PrarthonaPaul Just a total minor, since we are using StringBuilder with append calls here, we can remove the concatenation with `+`, so its better to use `combinedScopes.append(" ").append(scope);` here

Reply:
Fixed! 
Thank you @Skyllarr for the reviews. I have added the changes and squashed them to the same commit.